### ISSUE TITLE
Run buildkite on centos 7

### ISSUE THREAD

**Issue by tarekziade** [elastic/connectors#146]
### Problem Description

We need to support centos 7, let's make sure the service works there

### Proposed Solution

A new dockerfile cfore centos7 + python 3.7, used in the CI to run our tests, in parallel of the Python image

```
FROM centos:7

RUN yum update -y
RUN yum groupinstall "Development Tools" -y
RUN yum install openssl-devel libffi-devel bzip2-devel -y
RUN yum install wget -y

# open ssl
RUN wget https://www.openssl.org/source/openssl-3.0.5.tar.gz
RUN yum install perl-IPC-Cmd -y
RUN tar xzf openssl-3.0.5.tar.gz
RUN cd openssl-3.0.5 && ./config --prefix=/usr/local/custom-openssl --libdir=lib --openssldir=/etc/ssl && make -j1 depend && make -j8 && make install_sw

# python 3.10
RUN wget https://www.python.org/ftp/python/3.10.0/Python-3.10.0.tgz
RUN tar xvf Python-3.10.0.tgz
RUN cd Python-3.10.0
RUN cd Python-3.10.0 && ./configure --with-openssl=/usr/local/custom-openssl --with-openssl-rpath=auto --prefix=/usr/local/custom-openssl && make -j8 && make altinstall
RUN yum install -y python3-pip
RUN pip3 install certifi
RUN python3 -c 'import urllib.request;  print(urllib.request.urlopen("https://python.org/").status)'
```

**Comment by moxarth-elastic** [comment]
@tarekziade Thanks for providing the Dockerfile and it's working fine, so the dockerized image will work anyway in any environment. However, we were facing this problem while upgrading the already installed python 3.x version to python 3.10.x version in CentOS 7.
So, If user is installing python 3.10.x in fresh CentOS 7, it will work for them even without the docker.

**Comment by artem-shelkovnikov** [comment]
Additionally copied from another issue:

> We should test for CentOS 8 Stream

**Comment by seanstory** [comment]
@markjhoy is this issue still relevant, or is it duplicated by issues elsewhere?

**Comment by markjhoy** [comment]
My thought is we can probably just close this now as CentOs7 is no longer supported... CentOS 8, yes however
